src/components/displayBook.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import '../App.css';
+import { useSelector } from 'react-redux';
+import Display from './display';
+
+const DisplayBook = () => {
+  const books = useSelector((state) => state.booksReducer);
+  const result = [];
+  Object.keys(books).forEach((key) => {
+    const { title, author, id } = books[key];
+    result.push(<Display title={title} author={author} id={id} />);
+  });
+  return result;
```

Comment:
- [ ] OPTIONAL - Rather than this you can simply map through the books and return it:
```
return (
  books.map({title, author, id} => (<Book title={title} />))
)
```